changelog/0075-reindex-all-indexes-for-es-upgrade.yml

```diff
@@ -0,0 +1,85 @@
+title: reindex-all-indexes-for-es-upgrade
```

Comment:
nit: I think the `title` field can be more "human readable" right?

Reply:
I thought this is what is used in urls but I realised now that we have `key` for that. Fixed in [75743be](https://github.com/dimagi/commcare-cloud/pull/6151/commits/75743bef50a18f8fd230ff09d0440c7da26755c4)